protocols/mempool-overview.md

```diff
@@ -134,10 +139,134 @@ Committed transactions are not in the mempool:
 
 In blockchain, a tx is (or not) valid in a given state.
 That is a transaction can be valid (or not) at a given height of the ledger.
-To model this, consider that $p.ledger.valid(tx)$ is such a check for the current height of the ledger at process $p$ (ACBI call).
+To model this, consider that $p.ledger.valid(tx)$ is such a check for the current height of the ledger at process $p$ (ABCI call).
 Our third invariant is that only valid transactions are present in the mempool:  
 **INV3.** $\forall tx, \forall p \in Correct. \square(tx \in p.mempool \implies p.ledger.valid(tx))$
```

Comment:
Here, also, we need to add a footnote or something similar clarifying that operators may disable `ReCheckTx` via config change

Reply:
This paramter is now mentioned later in the text.